spec/furniture/marketplace/product_spec.rb

```diff
@@ -13,6 +13,14 @@
       product.price = 20
       expect(product.price_cents).to eql(2000)
     end
+
+    it('edits the price_cents') do
+      product = Marketplace::Product.new
+      product.price = 10
+      # TODO: FIX THIS TEST: edit the price via request response flow
```

Comment:
Do you want guidance on how to do this?  Or would you prefer to pair on it this Wednesday?

Reply:
Let's pair on it this Wednesday. I've drafted an outline of possible request specs for now https://github.com/zinc-collective/convene/pull/942/commits/1f9864160df8da6804027b8f4309390788d8c7da

Reply:
Nice!!!